.gitlab/common/system_probe.yml

```diff
@@ -0,0 +1,9 @@
+---
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
```